packages/web3-eth-accounts/package.json

```diff
@@ -28,7 +28,7 @@
 		"@types/jest": "^27.0.3",
 		"@typescript-eslint/eslint-plugin": "^5.4.0",
 		"@typescript-eslint/parser": "^5.4.0",
-		"eslint": "^8.3.0",
+		"eslint": "^7.32.0",
```

Comment:
Why this package is downgradded?

Reply:
This addresses this issue:

```bash
✔ Preparing...
⚠ Running tasks...
  ↓ No staged files match *.js [SKIPPED]
  ❯ Running tasks for *.ts
    ✔ prettier --write
    ✖ eslint [FAILED]
  ↓ No staged files match *.{json,md} [SKIPPED]
↓ Skipped because of errors from tasks. [SKIPPED]
✔ Reverting to original state because of errors...
✔ Cleaning up...

✖ eslint:

Oops! Something went wrong! :(

ESLint: 8.3.0

Error: Rules with suggestions must set the `meta.hasSuggestions` property to `true`. `meta.docs.suggestion` is ignored by ESLint.
```

I forgot I fixed that here, but will break it out into a separate PR

Reply:
[PR](https://github.com/ChainSafe/web3.js/pull/4628) that merges just these changes